### PR TITLE
style: cdn tiles

### PR DESCRIPTION
**Pull request by euhmeuh**

### CODE REVIEW

File: client/app/components/expiration/service-expiration-date.component.controller.js [hunk 1]
```diff
@@ -11,7 +11,7 @@
         }
 
         $onInit () {
-            const hasValidServiceInfos = !this.serviceInfos || angular.isObject(this.serviceInfos);
+            const hasValidServiceInfos = this.serviceInfos || angular.isObject(this.serviceInfos);
```
Comment (frenautvh) [inline client/app/components/expiration/service-expiration-date.component.controller.js]: then angular.isObject should be enough

File: client/app/cdn/dedicated/manage/cdn-dedicated-manage.controller.js [hunk 2]
```diff
@@ -20,9 +20,9 @@ angular.module("App").controller("CdnManageCtrl", class CdnManageCtrl {
         return this.$q.allSettled([
             this.Cdn.getSelected(this.$stateParams.productId, true),
             this.Cdn.getServiceInfos(this.$stateParams.productId)
-        ]).then((data) => {
-            this.cdn = data[0];
-            this.serviceInfos = data[1];
+        ]).then(([selected, serviceInfos]) => {
```
Comment (FredericEspiau) [inline client/app/cdn/dedicated/manage/cdn-dedicated-manage.controller.js]: This works ???

File: client/app/cdn/dedicated/manage/ssl/cdn-dedicated-manage-ssl.html [hunk 3]
```diff
@@ -25,31 +23,59 @@ <h2 data-translate="cdn_tab_ssl"></h2>
...
-                </dd>
-                <dt data-translate="cdn_configuration_ssl_certificateValidFrom"></dt>
-                <dd data-ng-bind="ssl.certificateValidFrom | date:'mediumDate'"></dd>
-                <dt data-translate="cdn_configuration_ssl_certificateValidTo"></dt>
-                <dd data-ng-bind="ssl.certificateValidTo | date:'mediumDate'"></dd>
-                <dt data-translate="cdn_configuration_ssl_certificateProvider"></dt>
-                <dd data-translate="{{ 'cdn_configuration_ssl_provider_' + ssl.certificateProvider }}"></dd>
-            </dl>
+            <div class="oui-tile mb-5 h-100">
+                <h4 class="oui-header_4 oui-tile__title"
```
Comment (FredericEspiau) [inline client/app/cdn/dedicated/manage/ssl/cdn-dedicated-manage-ssl.html]: `h1`, `h2`, and `h3` above ?

File: client/app/cdn/dedicated/manage/statistics/cdn-dedicated-manage-statistics.html [hunk 4]
```diff
@@ -1,161 +1,139 @@
...
-                                </div>
-                            </div>
-                        </li>
+                    <div class="oui-tile__item">
+                        <dl class="oui-tile__definition">
+                            <dt class="oui-tile__term" data-translate="cdn_configuration_activity"></dt>
+                            <dd class="oui-tile__description">
+                                <span data-ng-pluralize
+                                      data-count="$ctrl.cdn.activity"
+                                      data-when="{
```
Comment (FredericEspiau) [inline client/app/cdn/dedicated/manage/statistics/cdn-dedicated-manage-statistics.html]: Isn't there a better way to handle this with PascalPrecht ?

File: client/app/components/expiration/service-expiration-date.component.controller.js [hunk 5]
```diff
@@ -41,7 +41,7 @@
         }
 
         isAutoRenew () {
-            return this.serviceInfos.renew && (this.serviceInfos.renew.automatic || this.serviceInfos.renew.forced);
+            return this.serviceInfos && this.serviceInfos.renew && (this.serviceInfos.renew.automatic || this.serviceInfos.renew.forced);
```
Comment (FredericEspiau) [inline client/app/components/expiration/service-expiration-date.component.controller.js]: `_(this.serviceInfos).get("renew.automatic") || _(this.serviceInfos).get("renew.forced")`

File: client/app/cdn/dedicated/manage/cdn-dedicated-manage.controller.js (hunk 2 above)
Comment (antleblanc) [reply client/app/cdn/dedicated/manage/cdn-dedicated-manage.controller.js]: Destructuring, for sure.

File: client/app/cdn/dedicated/manage/statistics/cdn-dedicated-manage-statistics.html (hunk 4 above)
Comment (euhmeuh) [reply client/app/cdn/dedicated/manage/statistics/cdn-dedicated-manage-statistics.html]: ¯\_(ツ)_/¯

File: client/app/cdn/dedicated/manage/cdn-dedicated-manage.controller.js (hunk 2 above)
Comment (FredericEspiau) [reply client/app/cdn/dedicated/manage/cdn-dedicated-manage.controller.js]: In an array ? Not in an object ? I've never seen this before

**Pull request merged**